### DISCUSSION TITLE
How akhq is deserializing the messages that is serialized in java with default topic config

### DISCUSSION THREAD

**Discussion by ankurjains** [tchiotludo/akhq#851] [Q&A] [answered]
Hi I want to know how akhq deserialize messages that is being serilized in java . I can see all messages in string with json format my topic config are default and using 2.6-IV0 protocol version . ?

**Comment by tchiotludo** [comment, marked as answer]
Please add more detail if you want to have, help. 
A full reproduction is needed (a way to publish the same message) and a actual / wanted behaviour

**Reply by ankurjains** [reply]
I got it resolved
